### PR TITLE
Copy edit of text

### PR DESCRIPTION
**Pull request by Ainali**
This is a first pass of copy editing.

### CODE REVIEW

**Review by felixfaassen — APPROVED** [review]
great work @Ainali !

File: municipalities.md [hunk 1]
```diff
@@ -13,37 +13,37 @@ permalink: /municipalities
...
+The current information architecture makes it difficult for municipalities to securely exchange data in their own backoffice or between other municipalities. The current systems are lacking the flexibility to quickly adept and innovate in order to implement rapid change in public policy and serve the changing needs of its citizens. This makes maintaining and altering exisiting software increasingly slow and expensive resulting in a lower quality of service with higher costs. OpenZaak was developed to solve this problem helping municipalities increase their level of service and lower the cost of ownership.
```
Comment (ericherman) [inline municipalities.md]: s/quickly adept/quickly adapt/

File: municipalities.md [hunk 2]
```diff
@@ -13,37 +13,37 @@ permalink: /municipalities
...
 
-OpenZaak originally was developed by Maykin Media B.V. commissioned by Amsterdam, Rotterdam, Utrecht, Tilburg, Arnhem, Haarlem, 's-Hertogenbosch, Delft and a coalition of Hoorn, Medemblik, Stede Broec, Drechteland, Enkhuizen (SED), directed by [Dimpact](https://www.dimpact.nl/).
+OpenZaak was originally developed by Maykin Media B.V. and commissioned by Amsterdam, Rotterdam, Utrecht, Tilburg, Arnhem, Haarlem, 's-Hertogenbosch, Delft and a coalition of Hoorn, Medemblik, Stede Broec, Drechteland, Enkhuizen (SED) under direction of [Dimpact](https://www.dimpact.nl/).
```
Comment (ericherman) [inline municipalities.md]: I might choose "initially" rather than "originally", but either is okay.

Comment (Ainali) [reply municipalities.md]: I like initially, I'll change that.

File: vendors.md [hunk 3]
```diff
@@ -25,28 +25,28 @@ This means vendors can offer different types of services around OpenZaak. The mo
...
+We also invite you to [join and participate our community](/community) to learn how other municipalities and vendors are using OpenZaak. 
 
 ## How can I integrate OpenZaak into my product or service?
 
-The answer to this question largely depends on what kind of vendor you are and where you can offer the most value. Whether you are a full service IT vendor or software vendor. There are vendors who have decided to implement the ZGW API's from sratch and not relying on OpenZaak and there are vendors who integrated the OpenZaak codebased into their own product. Some vendors focus on hosting the back-end towards municipalities or even other vendors.
+The answer to this question largely depends on what kind of vendor you are and what value you want to offer, i.e. whether you are a full service IT vendor or software vendor. There are vendors who have decided to implement the ZGW API's from scratch and to not rely on OpenZaak and there are vendors who have integrated the OpenZaak codebased into their own product. Some vendors focus on hosting the back-end towards municipalities or even other vendors.
```
Comment (ericherman) [inline vendors.md]: Can we drop the _id est_ in this? There may be more types of vendors, the _i.e._ seems restrictive here.

**Review by ericherman — CHANGES_REQUESTED** [review]

File: vendors.md (hunk 3 above)
Comment (Ainali) [reply vendors.md]: Oh, I was actually going for e.g. but it slipped my mind. Would that be better or should we just skip it completely?

Comment (ericherman) [reply vendors.md]: yeah, "e.g." or "for instance" would work, but I think just skip it.

**Review by ericherman — APPROVED** [review]

**Pull request merged**